### ISSUE TITLE
Should model structure be evaluated in this paper

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/bp.delta#38]
Some of the various model structures discussed in #4 and others could be considered in the evaluation in this paper. This might add quite a lot of complexity but the potential pay-off might be more learning about how to incorporate strains in forecasting models

**Comment by seabbs** [comment]
Current options to include: are over dispersion and strain relationship

**Comment by jbracher** [comment]
I'm in favour of keeping things rather simple here. I could see a comparison to two simplifications of your current correlated random walks approach (completely correlated with constant ratio of R_t between variants, completely independent), but wouldn't get into e.g. dispersion .

**Comment by seabbs** [comment]
Yes broadly agree. The reason to keep dispersion at the moment is that the model has fitting issues with it present (because overdispersion competes with the random walk to deal with noise I think). If these are resolved as I look again at the model agree can drop. 

I think having a pooled model is quite important in order to in some way account for importation and non-random mixing across populations (I could also be totally wrong hence the interest in testing this assumption).
